client/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+
+
+function App() {
```

Comment:
small thing, but can we change this to 

```const App = () =>```

so that we can be consistent with using ES6 on the front end

:)